libraries/AP_Mount/AP_Mount_Xacti.cpp

```diff
@@ -228,7 +228,8 @@ void AP_Mount_Xacti::send_camera_information(mavlink_channel_t chan) const
         0,                      // lens_id uint8_t
         flags,                  // flags uint32_t (CAMERA_CAP_FLAGS)
         0,                      // cam_definition_version uint16_t
-        cam_definition_uri);    // cam_definition_uri char[140]
+        cam_definition_uri,     // cam_definition_uri char[140]
+        _instance + 1);      // gimbal_device_id uint8_t
```

Comment:
nitpick: slight white space issue on the comments.

Reply:
I missed it, sorry! Should be fixed now!